src/ontology/mondo-edit.obo

```diff
@@ -586025,6 +586025,16 @@ property_value: http://purl.org/dc/elements/1.1/date "2024-07-03T14:16:10Z" xsd:
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7719" xsd:anyURI
 
+[Term]
+id: MONDO:1040006
+name: CTR9-related neurodevelopmental disorder
+def: "Any neurodevelopmental disorder in which the cause of the disease is a variation in the CTR9 gene. This disorder is characterised by varying degrees of intellectual disability, neurodevelopmental delay, hypotonia, fatigability, behavioral abnormalities including autism spectrum disorder, anxiety and aggressive behavior, cardiac anomalies, and mild facial dysmorphism." [https://orcid.org/0000-0002-0587-4693, https://orcid.org/0000-0002-0987-9862, PMID:35468861, PMID:35499524, PMID:35717577]
+is_a: MONDO:0100500 {source="https://orcid.org/0000-0002-0587-4693", source="https://orcid.org/0000-0002-0987-9862"} ! Mendelian neurodevelopmental disorder
```

Comment:
I am adding the PMID as sources for this SubClass annotation as it straightened the evidence